flutter/lib/ui/benchmarks_configuration_screen.dart

```diff
@@ -54,14 +50,14 @@ class BenchmarksConfigurationScreen extends StatelessWidget {
   Widget build(BuildContext context) {
     final stringResources = AppLocalizations.of(context);
     final state = context.watch<BenchmarkState>();
-    final chosenConfiguration = state.chosenBenchmarksConfiguration;
+    final currentConfiguration = state.configManager.currentConfig;
```

Comment:
I would stick to the short form `Config` instead of `Configuration`

Reply:
done